test/templates.html

```diff
@@ -162,6 +165,24 @@
         it('should work inside header templates', () => {
           expect(getCellContent(getFirstHeaderCell(grid)).textContent).to.contain('fooheader1');
         });
+
+        it('should not invoke computed functions with null item', () => {
+          const spy = sinon.spy(container, '_formatItem');
+          grid.size = 1000;
+          grid._scrollToIndex(100);
+          spy.getCalls().forEach(call => {
+            expect(call.args[0]).not.to.be.null;
+          });
+        });
+
+        it('should not invoke computed functions with empty item', () => {
+          const spy = sinon.spy(container, '_formatItem');
+          grid.size = 1000;
+          grid._scrollToIndex(100);
```

Comment:
You may want to set the index to be higher. I only got this error when scrolling towards the end of a long list of items. Unless you were able to get this to fail before you put in the fix, in that case ignore :-)

Reply:
Both tests are failing without the fix. It's enough to scroll past the currently cached page (50 first items in this case)